src/ComplexityVisitor.js

```diff
@@ -26,18 +64,33 @@ export default class ComplexityVisitor {
     this.costFactor = 1;
 
     this.rootCalculator = new CostCalculator();
-    this.fragmentCalculators = Object.create(null);
 
     this.Field = {
       enter: this.enterField,
       leave: this.leaveField,
     };
 
-    this.FragmentSpread = this.enterFragmentSpread;
+    this.FragmentDefinition = () => {
```

Comment:
i suspect there's a default "don't do anything" handler, as this doesn't handle all the AST types

Reply:
the comment/docs around return values is wrong for how visitor handlers work, `false` means `skip` vs `undefined` which means do nothing in particular. without the handler i believe it'll process the fragment selections